### ISSUE TITLE
QuickHide settings troubles

### ISSUE THREAD

**Issue by Tutul-** [linuxgurugamer/QuickMods#14]
When I go to the settings page to mark as always visible some button, the change only work for the current view and, if I return to that view again, most of them are reset. The list of "mods button" is becoming increasingly long so I assume it failed to detect button and see them as new one each time.

by view I mean the KSC/VAB/SPH/vessel/etc...

**Comment by Alexsys** [comment]
True, quickhide doesnt save the settings, it just creates more and more new writes it seems
